CHANGELOG.md

```diff
@@ -6,6 +6,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [2.0.2] - 2024-04-10
+### Added Automatic adaptor docker image build
+### Changed Large message size handling
```

Comment:
🤣  I'm not sure what format you're using here, but it certainly isn't the [Keep a Changelog](https://keepachangelog.com/en/1.0.0/) formatting!

Also, the changelog is aimed at end users. They aren't going to get much information from this information.

There isn't that much that users would care about in this release, but there were some package updates, so I'd recommend putting something like...

```suggestion
### Fixed
- Updated dependencies to keep adaptor secure.
```